.gitignore

```diff
@@ -17,4 +17,5 @@ venv
 .tox/
 tox.output
 .gitignore
-tox.ini
\ No newline at end of file
+tox.ini
+*.un~
```

Comment:
🤔

Reply:
Vim undo file 😬

Reply:
This should really be in your global `.gitignore`..

There's poor precedent here though, so feel free to leave as is.

Reply:
@pietern reverted.